extensions/WSLExtension/Contracts/IProcessCreator.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using WSLExtension.Models;
+
+namespace WSLExtension.Contracts;
+
+/// <summary>
+/// Abstraction used to create processes throughout the WSL extension.
+/// </summary>
+public interface IProcessCreator
+{
+    /// <summary>
+    /// Creates and starts a new process without opening a new Window. Note:

```

Comment:
The name of the function says "WithWindow". The comment says "without opening a new Window".

And it's in reverse for CreateProcessWithoutWindow method below.

Reply:
will fix, copy/paste error